.snippets/code/tutorials/integrations/local-subsquid/hardhat-config.js

```diff
@@ -0,0 +1,17 @@
+// Import the Ethers plugin required to interact with the contract
+require('@nomicfoundation/hardhat-ethers');
+
+/** @type import('hardhat/config').HardhatUserConfig */
+module.exports = {
+  // Specify the Solidity version
+  solidity: '0.8.20',
+  networks: {
+    dev: {
+      url: 'http://127.0.0.1:9944',
+      chainId: 1281, // (hex: {{ networks.development.hex_chain_id }}),
```

Comment:
the variables don't render in snippets
```suggestion
      chainId: 1281, // (hex: 0x501),
```